src/grading-settings/assignment-section/index.jsx

```diff
@@ -0,0 +1,223 @@
+              <AssignmentItem
+                className="course-grading-assignment-total-grade"
+                title={intl.formatMessage(messages.weightOfTotalGradeTitle)}
+                descriptions={intl.formatMessage(messages.weightOfTotalGradeDescription)}
+                type="number"
+                min={MIN_NUMBER_VALUE}
+                max={MAX_NUMBER_VALUE}
+                errorMsg={intl.formatMessage(messages.weightOfTotalGradeErrorMessage)}
+                name={weight}
+                value={gradeField.weight}
+                onChange={(e) => handleAssignmentChange(e, idx)}
+                errorEffort={errorList[`${weight}-${idx}`]}
+              />
+              <AssignmentItem
+                className="course-grading-assignment-total-number"
+                title={intl.formatMessage(messages.totalNumberTitle)}
+                descriptions={intl.formatMessage(messages.totalNumberDescription)}
+                type="number"
+                min={1}
+                errorMsg={intl.formatMessage(messages.totalNumberErrorMessage)}
+                name={minCount}
+                value={gradeField.minCount}
+                onChange={(e) => handleAssignmentChange(e, idx)}
+                errorEffort={errorList[`${minCount}-${idx}`]}
+              />
+              <AssignmentItem
+                className="course-grading-assignment-number-droppable"
+                title={intl.formatMessage(messages.numberOfDroppableTitle)}
+                descriptions={intl.formatMessage(messages.numberOfDroppableDescription)}
+                type="number"
+                min={MIN_NUMBER_VALUE}
+                errorMsg={intl.formatMessage(messages.numberOfDroppableErrorMessage)}
+                name={dropCount}
+                gradeField={gradeField}
+                value={gradeField.dropCount}
+                onChange={(e) => handleAssignmentChange(e, idx)}
+                secondErrorMsg={(
+                  <FormattedMessage
+                    id="course-authoring.grading-settings.assignment.number-of-droppable.second.error.message"
+                    defaultMessage="Cannot drop more {type} assignments than are assigned."
+                    values={{ type: gradeField.type }}
+                  />
+                )}
+                errorEffort={errorList[`${dropCount}-${idx}`]}
+              />
+            </ol>
+            {gradeField.minCount !== courseAssignmentUsage?.length && Boolean(courseAssignmentUsage?.length) && (
+              <AlertMessage
+                className="course-grading-assignment-item-alert-warning"
+                variant="warning"
+                icon={Warning}
+                title={(
+                  <FormattedMessage
+                    id="course-authoring.grading-settings.assignment.alert.warning.usage.title"
+                    defaultMessage="Warning: The number of {type} assignments defined here does not match the current number of {type} assignments in the course:"
```

Comment:
the same here as above

Reply:
<img width="1227" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/5ff6d2f3-0c1f-40dd-917d-195d6660bc8a">

Reply:
Added